doc/psidk/src/run-infrastructure/cli/psibase.md

```diff
@@ -123,6 +124,20 @@ Deploy a service
 
   Sender to use when creating the account [default: account-sys]
 
+### info
+
+`psibase` [`-a` *url*] `info` *packages*\.\.\.  
+
+Displays the contents of packages
+
+- *packages*
+
+  Packages to show
+
+- `--package-source` *url*
+
+  Specifies a package repository. If multiple repositories are provided, the ones listed earlier will be preferred over those listed later. The default is the local package repository.
```

Comment:
What is a valid package repository url? What is the local package repository url? What makes a package repo url valid? I have no idea what URL can be put here

Reply:
It could also be clearer that `share/psibase/packages` is permitted. I wouldn't expect that given that it says, `*url*`.

Reply:
In practice, I would only expect to see a local path for internal testing.